tools/dart/update.py

```diff
@@ -73,8 +74,11 @@ def main():
     with zipfile.ZipFile(output_file, 'r') as zip_ref:
```

Comment:
Small cleanup suggestion: Maybe pull some or all of the zipfile extraction into a helper function.